tardis/plasma/base.py

```diff
@@ -316,8 +316,9 @@ def write_to_dot(self, fname, args=None, latex_label=True):
 
         for edge in print_graph.edges:
             label = print_graph.edges[edge]["label"]
-            print_graph.edges[edge]["label"] = "-"
-            print_graph.edges[edge]["texlbl"] = label
+            print_graph.edges[edge]["label"] = " "
+            if latex_label:
```

Comment:
`if latex_label is not None:` or something similar.

Reply:
since `latex_label` is a boolean, I opted to change the if statements to be `if latex_label == True` to show that it is a boolean